src/components/cladeFilterDrawer/samplesOfInterest.tsx

```diff
@@ -5,46 +5,62 @@ import Button from "@mui/material/Button";
 import Tooltip from "@mui/material/Tooltip";
 import { tooltipProps } from "../formatters/sidenote";
 import InfoOutlinedIcon from "@mui/icons-material/InfoOutlined";
+import { FormControl, FormControlLabel, FormLabel } from "@mui/material";
+import AddIcon from "@mui/icons-material/Add";
+import { useState } from "react";
+import CaseDefinitionConstructor from "./caseDefinitionConstructor";
 
 export const SamplesOfInterest = () => {
   const dispatch = useDispatch();
   // @ts-ignore -- TODO: figure out how to add types to state
   const state = useSelector((state) => state.global);
 
+  const [caseDefOpen, setCaseDefOpen] = useState<boolean>(false);
+
   return (
     <div>
-      <h2>
-        Samples of interest{" "}
-        <Tooltip
-          title={`Specify samples of interest -- either by name or using a case definition
-          (below) -- to see them highlighted in the graph and filter to clades
-          that contain at least one of these samples.`}
-          componentsProps={tooltipProps}
-          arrow
-        >
-          <InfoOutlinedIcon
-            sx={{
-              position: "relative",
-              top: 5,
-              // color: "#4f2379",
-              fontSize: 20,
+      <h2>Filter to clades which contain your samples of interest </h2>
+      <p>
+        <FormLabel>
+          Add samples of interest to see them highlighted in the graph and
+          filter to clades which contain at least one of these samples.
+        </FormLabel>
+      </p>
+
+      <div
+        style={{
+          display: "flex",
+          flexDirection: "row",
+          justifyContent: "space-between",
+        }}
+      >
+        <div>
+          <h4 style={{ marginBottom: 0, marginTop: 0 }}>Sample Names: </h4>
+          <FormLabel>
+            {state.samplesOfInterest.length} Samples of interest
+          </FormLabel>
+        </div>
+        <div style={{ position: "relative", top: 20 }}>
+          <Button
+            variant="text"
+            color="primary"
+            disabled={state.samplesOfInterestNames === ""}
+            onClick={() => {
+              dispatch({ type: "sample names string changed", data: "" });
```

Comment:
This is pre-existing, but I would like to stat moving all the redux action types into constants or an enum or something, so we can be sure they match up across the app and there is a single source of truth for seeing all the possible actions in one place. Don't worry about going back and changing the pre-existing ones, but for new actions I think we should start doing that, and eventually the legacy actions can get rolled into it in the future.

Reply:
Sure thing! Can you make an example one? Happy to pattern match.